From: Markus Koschany <apo@gambaru.de>
Date: Thu, 26 Feb 2015 22:16:36 +0100
Subject: fix attach so path

Adjust the path to attach_linux.so since the location is different in Debian.

Forwarded: not-needed
---
 .../pysrc/pydevd_attach_to_process/add_code_to_python_process.py      | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/plugins/org.python.pydev/pysrc/pydevd_attach_to_process/add_code_to_python_process.py b/plugins/org.python.pydev/pysrc/pydevd_attach_to_process/add_code_to_python_process.py
index d8dca95..723a524 100644
--- a/plugins/org.python.pydev/pysrc/pydevd_attach_to_process/add_code_to_python_process.py
+++ b/plugins/org.python.pydev/pysrc/pydevd_attach_to_process/add_code_to_python_process.py
@@ -414,7 +414,7 @@ def run_python_code_linux(pid, python_code, connect_debugger_tracing=False, show
     print('Attaching with arch: %s'% (arch,))
 
     target_dll = os.path.join(filedir, 'attach_linux_%s.so' % suffix)
-    target_dll = os.path.normpath(target_dll)
+    target_dll = os.path.normpath('/usr/lib/eclipse-pydev/attach_linux.so')
     if not os.path.exists(target_dll):
         raise RuntimeError('Could not find dll file to inject: %s' % target_dll)
 
@@ -437,8 +437,6 @@ def run_python_code_linux(pid, python_code, connect_debugger_tracing=False, show
 
     cmd.extend(["--eval-command='set scheduler-locking off'"])  # If on we'll deadlock.
 
-    cmd.extend(["--eval-command='set architecture %s'" % arch])
-
     cmd.extend([
         "--eval-command='call dlopen(\"%s\", 2)'" % target_dll,
         "--eval-command='call DoAttach(%s, \"%s\", %s)'" % (
